ansible/roles/ocp-workload-fsi-client-onboarding-demo/defaults/main.yml

```diff
@@ -23,3 +23,8 @@ build_status_delay: 20
 
 deploy_status_retries: 15
 deploy_status_delay: 20
+
+
+bpms_version_tag: 6.4.x
+bpms_imagestreams_yml: https://raw.githubusercontent.com/jboss-container-images/jboss-processserver-6-openshift-image/{{bpms_version_tag}}/templates/processserver64-image-stream.json"
+bpms_imagestreams_tag: 1.0
```

Comment:
If applicable, can you please use the full qualified variable names ?

  for example:

- `ocp_workload_fsi_client_onboarding_demo_bpms_version_tag`
-  `deploy_status_delay`

This prevent collisions.

Reply:
i meant  for second var:

-  `ocp_workload_fsi_client_onboarding_demo_deploy_status_delay`